.gitignore

```diff
@@ -127,3 +127,4 @@ integration_tests/videos/
 integration_tests/screenshots/
 */*.iml
 **/Chart.lock
+.DS_Store
```

Comment:
The preferred way of ignoring editor artefacts, toolchain log files, OS files and so on is to include these in a global .gitignore file and not part of the code repository.